apps/AEPSampleApp/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,8 +15,10 @@
 
         <category android:name="android.intent.category.DEFAULT" />
         <category android:name="android.intent.category.LAUNCHER" />
-        <category android:name="android.intent.category.BROWSABLE" />
 
+      </intent-filter>
```

Comment:
Can we revert this change? Do we have a reason to have two `<intent-filter>` sections?

Reply:
we need to have this change for the app to run on API 33.  The current setup gets error when running on API 33.  With this change, the app can be run on all Android API versions.

Reply:
But, we already have the `<category android:name="android.intent.category.BROWSABLE" />` configuration. Why should we remove it and add it in another `<intent-filter>` section?

Reply:
It is about we need to move Deeplink setup into another intent-filter section to resolve the error when running on an API33 device. If not, we will get an error.
Error type 3
Error: Activity class does not exist. 

And while moving 
<data android:scheme="assurance" /> 
into another <intent-filter> section, it gives format error for <category android:name="android.intent.category.BROWSABLE" /> 
I need to move <category android:name="android.intent.category.BROWSABLE" /> into the next <intent-filter> to resolve the format error.

To keep the setup similar to the original AndroidManifest file and also work with API 33, the current code was what I tried and worked.

Reply:
Can we remove the `<data android:scheme="assurance" />` and see if it works with one `<intent-filter>`? I don't think we need the above "assurance" configuration for the sample app.